app/Resources/views/layout.html.twig

```diff
@@ -122,7 +122,7 @@ along with Incipio as the file LICENSE.  If not, see <http://www.gnu.org/license
             {# It is illegal (cf licence) to remove the link to github #}
             {# And dishonest to remove the reference to Jeyser's main contributors.#}
             <p class="text-muted" style="text-align: center">
-                <i>© Jeyser CRM 2017 &#124; N7 Consulting - AB</i>
+                <i>© Jeyser CRM 2017 &#124; M-GaTE - OR &#124; N7 Consulting - AB</i>
```

Comment:
If we were only 2 contributors I would agree with that change.

However, if each and ever one of contributors add its letters, the footer will be too big. What about reducing it to 
`... M-GaTE | N7 Consulting` ?